docs/pages/router/basics/navigation.mdx

```diff
@@ -0,0 +1,256 @@
+
+## Native navigation basics with `useRouter`
+
+Like in React Navigation, you can call a function from an `onPress` handler to navigate to another page. In Expo Router, you can use the `useRouter` hook to access navigation functions:
+
+```tsx
+import { useRouter } from 'expo-router';
+
+export default function Home() {
+  const router = useRouter();
+
+  return <Button onPress={() => router.navigate('/about')}>Go to About</Button>;
+}
+```
+
+Expo Router apps default to stack navigation, where navigating to a new route pushes a screen onto a stack, and backing out of that route pops it off the stack. Usually, you would want to use the `router.navigate` function. This will either push a new page onto the stack or unwinding to an existing route on the stack. However, you can also call `router.push` to explicitly push a new page onto the stack, `router.back` to go back to the previous page, or `router.replace` to replace the current page on the stack.
+
+With Expo Router, you refer to pages by their URL, or their position relative to the **app** folder. Check out the following file structure and how you would navigate to each page:
+
+<FileTree
+  files={[
+    ['app/index.tsx', '`router.navigate("/")`'],
+    ['app/about.tsx', '`router.navigate("/about")`'],
+    ['app/profile/index.tsx', '`router.navigate("/profile")`'],
+    ['app/profile/friends.tsx', '`router.navigate("/profile/friends")`'],
+  ]}
+/>
+
+## Links and Buttons
+
+However, the typical way to link to a page in Expo Router is to use links, just like web apps. Expo Router has `Link` components for navigating between pages, where the `href` is the same route you would use in `router.navigate`:
+
+{/* prettier-ignore */}
+```tsx app/index.tsx
+import { View } from 'react-native';
+/* @info Import the <strong>Link</strong> React component from <strong>expo-router</strong> */
+import { Link } from 'expo-router';
+/* @end */
+
+export default function Page() {
+  return (
+    <View>
+      /* @info Tapping this will link to the <strong>about</strong> page */
+      <Link href="/about">About</Link>
+    </View>
+  );
+}
+```
+
+By default, Links can only wrap `Text` components. You can use `Pressable` or other components that support `onPress` and `onClick` props inside a link with the `asChild` prop:
+
+{/* prettier-ignore */}
+```tsx
+import { Pressable, Text } from 'react-native';
+import { Link } from 'expo-router';
+
+export default function Page() {
+  return (
+    /* @info The <strong>onPress</strong> event that navigates to <strong>/other</strong> will be passed to <strong>Pressable</strong> */
+    <Link href="/other" asChild>
+    /* @end */
+      <Pressable>
+        <Text>Home</Text>
+      </Pressable>
+    </Link>
+  );
+}
+```
+
+## Relative routes
+
+You don't always have to use the absolute path to a route. Using paths that start with `./` (for the current folder) or `../` (for the parent folder) will navigate relative to the current route.
```

Comment:
I didn't know you could do that! I notice that `<Link>` also has a `relativeToDirectory` prop. I'm curious what the best practice is for navigating using a relative path? @marklawlor